v2/margin_service.go

```diff
@@ -726,6 +733,7 @@ func (s *GetMaxBorrowableService) Do(ctx context.Context, opts ...RequestOption)
 		secType:  secTypeSigned,
 	}
 	r.setParam("asset", s.asset)
+	r.setParam("isolatedSymbol", s.isolatedSymbol)
```

Comment:
Hi C-ollins! I am please to see your pull request because I am waiting for this enhancement as well.

I reviewed your code and I think that we need to check if the parameter has been set:

```suggestion
	if s.isolatedSymbol != "" {
		r.setParam("isolatedSymbol", s.isolatedSymbol)
	}
```

Besides, this will make the unit test pass.